server_http.hpp

```diff
@@ -146,6 +146,15 @@ namespace SimpleWeb {
         write(StatusCode::success_ok, std::string(), header);
       }
 
+      void write_more(const std::string &content) {
+        *this << content;
+      }
+
+      void set_nodelay() {
+        asio::ip::tcp::no_delay option(true);
+        session->connection->socket->set_option(option);
```

Comment:
Thank you, but the no_delay option is already set here: https://github.com/eidheim/Simple-Web-Server/blob/master/server_http.hpp#L712

Reply:
Ah! I missed that.